src/main/java/net/geforcemods/securitycraft/tileentity/ReinforcedButtonTileEntity.java

```diff
@@ -0,0 +1,26 @@
+package net.geforcemods.securitycraft.tileentity;
+
+import net.geforcemods.securitycraft.SCContent;
+import net.geforcemods.securitycraft.api.CustomizableTileEntity;
+import net.geforcemods.securitycraft.api.Option;
+import net.geforcemods.securitycraft.misc.CustomModules;
+
+public class ReinforcedButtonTileEntity extends CustomizableTileEntity
+{
+	public ReinforcedButtonTileEntity()
+	{
+		super(SCContent.teTypeReinforcedButton);
+	}
+
+	@Override
+	public CustomModules[] acceptedModules()
+	{
+		return new CustomModules[] {CustomModules.WHITELIST};
+	}
+
+	@Override
+	public Option<?>[] customOptions()
+	{
+		return null;
+	}
+}
```

Comment:
Remove this tile entity, rename the tile entity for the pressure plates to `WhitelistOnlyTileentity` and adapt the tile entity types. Make sure to not change the registry name of the pressure plate tile entity because doing that will break existing worlds.

Reply:
You need to add all the blocks from the reinforced_button te type registration in RegistrationHandler to the one from reinforced_pressure_plate.